kernel/syno-armada375-6.2.3/Makefile

```diff
@@ -0,0 +1,22 @@
+ARCH ?= armada375
+TCVERSION ?= 6.2.3
+PKG_NAME = armada375
+PKG_BRANCH = 25426
+PKG_ARCH = $(PKG_NAME)
+PKG_EXT = txz
+PKG_DIST_NAME = linux-3.x.$(PKG_EXT)
+PKG_DIST_SITE = https://downloads.sourceforge.net/project/dsgpl/Synology%20NAS%20GPL%20Source/$(PKG_BRANCH)branch/$(PKG_ARCH)-source
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_DIST_NAME)
+PKG_EXTRACT = linux-3.x
+SYNO_CONFIG = synoconfigs/$(PKG_ARCH)
+BASE_ARCH = arm
+
+HOMEPAGE = http://www.synology.com/
+
+PRE_COMPILE_TARGET = myPreCompile
```

Comment:
I propose a more explicit target name like for instance `kernel_armada375_platv2`

Reply:
I'm not sure I follow?
BTW, kernel is the only target I haven't "simplified" the Makefiles yet (vs `toolchain` and `toolkit`)
On my todo list for a later time.

Reply:
@th0ma7 please do this little housekeeping now, the `myPreCompile` naming is an influenza that still has its traces every where.
my proposal is to name the target `kernel_armada375_pre_compile`